python/dgl/distributed/dist_graph.py

```diff
@@ -928,10 +885,8 @@ def num_edges(self, etype=None):
         123718280
         """
         if etype is None:
-            if len(self.etypes) == 1:
-                return self._gpb._num_edges(self.etypes[0])
-            else:
-                return sum([self._gpb._num_edges(etype) for etype in self.etypes])
+            return sum([self._gpb._num_edges(etype) \
+                for etype in self.canonical_etypes])
```

Comment:
The variable shadowing makes it quite difficult to read. Could you change the iter variable inside the list comprehension to sth else?

Reply:
Also, I don't think you need to add `\` if you use formatting tools like black.